Cargo.toml

```diff
@@ -65,6 +65,7 @@ actix-web               = "4"
 aes-gcm                 = "0.10"
 alloy                   = "0.11"
 anyhow                  = "1"
+assert-json-diff        = "0"
```

Comment:
@penso it looks like the latest version of this package is `v2.0.2`. any reason to choose v0?

Reply:
none, let me update it to 2

Reply:
@penso i just did a commit to update all dependencies to latest (https://github.com/left-curve/left-curve/commit/6e52d92cd106c695df64a219d0166a6276a202c6) including bumping this one to 2, so no need for you to do anything